### ISSUE TITLE
Editor component does not work at all

### ISSUE THREAD

**Issue by ga676005** [primefaces/primevue#1774]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primevue/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=110
```

**Minimal reproduction of the problem with instructions**
doc page
https://primefaces.org/primevue/showcase/#/editor

demo link from the doc page
https://codesandbox.io/s/o0pri

Editor is not rendered
![image](https://user-images.githubusercontent.com/51456572/142364243-85ec165c-b2ee-486e-a648-c76903e363e7.png)

**Comment by tugcekucukoglu** [comment]
Please make sure that Editor component is imported in main.js

**Comment by ga676005** [comment]
Hi, my original issue was this error shown in the image below but I found the demo page not working so I thought it was the issue. Should I open a new issue?

![image](https://user-images.githubusercontent.com/51456572/142382207-4e4aed74-34ee-4c07-b09a-c94681eaaeca.png)


I have quill installed by copying `npm install quill --save` on the doc, and simply add this example into my code, then I got the error.

```vue
<Editor v-model="value" editorStyle="height: 320px"/>
```


There are a few components cannot work properly when auto-imported by `unplugin-vue-components`, I am not sure if `Editor` is one of those since it does not require registering other services, but it has a dependency. Manually register `Editor` globally in `main.js` does not work in this case, it seems that `unplugin-vue-components` have higher control on importing components. So those components did not work properly had been removed from the auto-imported list.

dependencies
```json
"vue": "^3.2.22",
"vite": "^2.6.14",
"unplugin-vue-components": "^0.17.2",
"primevue": "^3.8.2",
"quill": "^1.3.7",
```
